capplets/keyboard/ui-a11y.suppr

```diff
@@ -1,3 +1,5 @@
+mate-keyboard-properties-a11y-notifications.ui://GtkLabel[@id='label1'] orphan-label
+mate-keyboard-properties-a11y-notifications.ui://GtkLabel[@id='label2'] orphan-label
```

Comment:
This is not nice indeed, but I plan on rewriting that UI file later on not to have all these issues.